.github/pull_request_template.md

```diff
@@ -1,25 +1,28 @@
 TODO: Please describe your changes here.
 
-TODO: Delete the following lines but _keep and fill out_ (!) the checklist below.
+<!--
+TODO: Please read the following on ICU Contributing
 
-Thank you for your pull request!\
-General info on contributing: Please see https://github.com/unicode-org/icu/blob/main/CONTRIBUTING.md
+Thank you for your pull request! 
 
-Contributors license agreement (CLA):
-- You will be automatically asked to sign the CLA before the PR is accepted.
-- To sign the CLA: https://cla-assistant.io/unicode-org/icu
+* For general info on contributing: https://github.com/unicode-org/icu/blob/main/CONTRIBUTING.md
+* Associating PRs to Jira issues
+  - We require each pull request to be associated with a Jira issue.
+  - Reuse existing issues for minor changes:
+    * ICU 77 docs minor fixes: ICU-22921 — User Guide & API docs typos etc., and version updates (e.g., dependabot for User Guide)
+    * ICU 77 code warnings/version updates: ICU-22920 — Fix compiler warnings. Update versions of code-related dependencies (e.g., dependabot).
+* Contributors license agreement (CLA):
+  - You will be automatically asked to sign the CLA before the PR is accepted.
+  - To sign the CLA: https://cla-assistant.io/unicode-org/icu
+  - For terms of use and license, see https://www.unicode.org/terms_of_use.html
+-->
 
-For terms of use and license, see https://www.unicode.org/terms_of_use.html
-
-TODO: We require each pull request to be associated with a Jira issue.
-- For minor changes you can use one of the following per-release tickets:
-- ICU 77 code warnings/version updates: ICU-22920 — Fix compiler warnings. Update versions of code-related dependencies (e.g., dependabot).
-- ICU 77 docs minor fixes: ICU-22921 — User Guide & API docs typos etc., and version updates (e.g., dependabot for User Guide)
-
-TODO: End of lines to delete.
+<!--
+TODO: Fill out the checklist below
+-->
 
 #### Checklist
-- [ ] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-NNNNN
+- [ ] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-_____
```

Comment:
Separately: We could remove the `https://unicode-org.atlassian.net/browse/` prefix because ICU-number auto-links. CLDR does this.

Reply:
Done.